### PR TITLE
Moc globe atl Examples

### PR DESCRIPTION
**Pull request by erogluorhan**
This PR adds an example script for moc_globe_atl function as follows: 

**docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py**

This example script reads a netCDF file ("**tavg_downsized.nc**") in, generates the function parameters from variables of this netCDF file, and runs moc_globe_atl function on these parameters. 

Because no plotting is performed within this example, it is intentionally located under the directory:

**docs/_static/scripts/examples/comp_only**

instead of **docs/_static/scripts/examples/matplotlib** directory.

### CODE REVIEW

File: docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# coding: utf-8
+""" This example scrit demonstrates how to computes POP MOC field offline from POP netcdf history files.  This routine
```
Comment (clyne) [inline docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py]: Typo: "scrit" -> "script"

File: docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+a_wvel   = np.where(ocean, w_e[0,:,:,:] * tarea3d, 0.0)
+a_bolus   = np.where(ocean, w_i[0,:,:,:] * tarea3d, 0.0)
+a_submeso   = np.where(ocean, w_sm[0,:,:,:] * tarea3d, 0.0)
+
+# Invoke `linint2` from `geocat-comp`
+result = geocat.comp.moc_globe_atl(lat_aux_grid, a_wvel, a_bolus, a_submeso, tlat, rmlak, msg=None, meta=False)
+
+print("moc_globe_atl successfully generated output.")
+
+asd = 6
```
Comment (clyne) [inline docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py]: What is this?

asd = 6

File: docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py (hunk 1 above)
Comment (erogluorhan) [reply docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py]: Fixed with a new push.

File: docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py (hunk 2 above)
Comment (erogluorhan) [reply docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py]: Was nothing more than a dummy breakpoint and forgotten there. Removed with a new push.

File: docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+k3d = np.repeat(np.repeat(np.arange(0, km, 1).reshape(km,1), ny, axis=1)[:,:,np.newaxis], nx, axis=2)
+kmt3d = np.repeat(kmt.values[np.newaxis, :, :], km, axis=0)
+tarea3d = np.repeat(tarea.values[np.newaxis, :, :], km, axis=0)
+ocean   = k3d <= kmt3d
+
+a_wvel   = np.where(ocean, w_e[0,:,:,:] * tarea3d, 0.0)
+a_bolus   = np.where(ocean, w_i[0,:,:,:] * tarea3d, 0.0)
+a_submeso   = np.where(ocean, w_sm[0,:,:,:] * tarea3d, 0.0)
+
+# Invoke `linint2` from `geocat-comp`
```
Comment (khallock) [inline docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py]: Should be
```
# Invoke `moc_globe_atl` from `geocat-comp`
```

Comment (erogluorhan) [reply docs/_static/scripts/examples/comp_only/example_moc_globe_atl.py]: Done

**Comment by khallock** [comment]
The checks for this are still intermittently failing due to a bug in `dpres_plevel`, which is a known but unrelated issue. As far as I'm concerned, this PR can be merged since it only adds an example script and documentation, and it should be entirely unaffected by the `dpres_plevel` bug.

**Pull request merged**